kits/cpp/simple-transpiled/main.cpp

```diff
@@ -23,10 +23,10 @@ int main()
     
     /** AI Code Goes Below! **/
 
-    Player player = gameState.players[gameState.id];
-    Player opponent = gameState.players[(gameState.id + 1) % 2];
+    Player &player = gameState.players[gameState.id];
+    Player &opponent = gameState.players[(gameState.id + 1) % 2];
 
-    GameMap gameMap = gameState.map;
+    GameMap &gameMap = gameState.map;
```

Comment:
thanks for all the "pass by reference" changes

my c++ is highkey rusty 😅

Reply:
Oh yeah, no worries. I think this code is unlikely to cause people issues, but also I didn't want to give people the false impression that this is a cheap datastructure that they should copy around without consequences.